podium-gateway/src/main/webapp/app/shared/request/request-review-feedback.ts

```diff
@@ -12,8 +12,12 @@ import { RequestReviewDecision } from './request-review-decision';
 
 export class RequestReviewFeedback {
 
-    decision?: RequestReviewDecision;
-    feedback?: string;
+    id?: string;
+    advice?: RequestReviewDecision;
+    reviewer?: string;
```

Comment:
What's are you going to put here? Reviewer's name or is there any plan to also make it as User object?